cryptographic_estimators/SDEstimator/SDWorkfactorModels/bjmm.py

```diff
@@ -70,37 +66,45 @@ def _build_model_and_set_constraints(self):
         ]
 
     def _memory(self, x):
-        """
-        max memory = max over each list
+        """Computes the maximum value across three lists.
+
+        Args:
+            x: Input to be used in the computations of the three lists.
+
+        Returns:
+            The maximum value among the results of self.L1(x), self.L2(x), and self.L3(x).
         """
         return max(self.L1(x), self.L2(x), self.L3(x))
 
-    def _time_lists(self, x: float):
-        """
-        time to construct each list = max(size input list, size output list)
+    def _time_lists(self, x):
+        """Compute the time to construct each list.
+
+        The time to construct each list is the maximum of the size of the input
+        list and twice the size of the input list minus a constant.
```

Comment:
Should be 
 The time to construct the lists is taken as the maximum of the involved lists sizes, i.e., the input and output lists.